tests/certificate-with-password.pem

```diff
@@ -0,0 +1,51 @@
+-----BEGIN ENCRYPTED PRIVATE KEY-----
```

Comment:
Does msal Python utilize CI pipeline? Can you get these test certs from the lab instead of committing them here?

Reply:
Those certs are not certs for any real services. We just created some dummy certs for testing the cert loading code path. Thanks for your review, though. :)